apache/spark
# [SPARK-8731] Beeline doesn't work with -e option when started in background

### What changes were proposed in this pull request?
Append jline option "-Djline.terminal=jline.UnsupportedTerminal" to  enable the Beeline process to run in background.

### Why are the changes needed?
Currently, if we execute spark Beeline in background, the Beeline process stops immediately.
<img width="1350" alt="image" src="https://user-images.githubusercontent.com/88070094/194742935-8235b1ba-386e-4470-b182-873ef185e19f.png">

### Does this PR introduce _any_ user-facing change?
User will be able to execute Spark Beeline in background. 

### How was this patch tested?

1. Start Spark ThriftServer
2. Execute command `./bin/beeline -u "jdbc:hive2://localhost:10000" -e "select 1;" &`
3. Verify Beeline process output in console:
<img width="1407" alt="image" src="https://user-images.githubusercontent.com/88070094/194743153-ff3f1d19-ac23-443b-97a6-f024719008cd.png">

### Note

Beeline works fine on Windows when backgrounded:
![image](https://user-images.githubusercontent.com/88070094/194743797-7dc4fc21-dec6-4056-8b13-21fc96f1476e.png)
